dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -70,6 +70,36 @@ public void resetDBBetweenTests() throws Exception {
         CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
     }
 
+    @Test
+    public void refreshAll() {
+        // get userid
+        final long userid = testingPostgres.runSelectStatement(String.format("SELECT id FROM user_profile WHERE username='%s';", USER_2_USERNAME), long.class);
+
+        // Make sure there are no entries associated with this user
+        testingPostgres.runDeleteStatement(String.format("DELETE FROM user_entry ue WHERE ue.userid = %d", userid));
+
+        // refresh all by organizations (We are testing the CLI implementation of this refresh functionality, we do the refresh here to give a baseline)
+        refreshByOrganizationReplacement(USER_2_USERNAME);
+
+        // initial count of workflows associated with this user using the standard refresh
+        final long val1 = testingPostgres.runSelectStatement(String.format("SELECT COUNT(*) FROM user_entry WHERE userid = %d;", userid), long.class);
+
+        // Delete all user_entry rows associated with repos this user is added to
+        final int deleteResult = testingPostgres.runDeleteStatement(String.format("DELETE FROM user_entry ue WHERE ue.userid = %d", userid));
+        assertEquals("After deletion, there should be 0 rows remaining associated with this user", deleteResult, val1);
```

Comment:
I'm not sure you need line 85. Or, alternatively, maybe you don't need line 88.

If I'm reading this right, You've already deleted all the user entries on line 79. So I think you just want to assert that there are 0 entries associated with the user.

BTW, we're not consistent on this, but the second parameter in this case is the expected value, but you have it as the actual value. While it will fail either way if they are off, the message that is printed out makes it good to have the params in the correct order.

Reply:
I've updated the flow of the test case to be more direct. Workflows are initially registered, then all entries associated with the test user are deleted, refreshed, and checked for correctness.

Incorrect orderings of expected/actual parameters has been fixed.